solutions/security/ai/use-cases.md

```diff
@@ -18,6 +18,12 @@ The guides in this section describe example workflows for AI Assistant and Attac
 
 For general information, refer to [AI Assistant](/solutions/security/ai/ai-assistant.md) or [Attack discovery](/solutions/security/ai/attack-discovery.md).
 
+## Other AI-powered tools
 
+In addition to AI Assistant and Attack Discovery, {{elastic-sec}} provides several other AI-powered tools for specific use-cases. These include:
+
+* [Automatic Import](/solutions/security/get-started/automatic-import.md): helps you quickly parse, ingest, and create [ECS mappings](https://www.elastic.co/elasticsearch/common-schema) for data from sources that don’t yet have prebuilt Elastic integrations. This can accelerate your migration to {{elastic-sec}}, and help you quickly add new data sources to an existing SIEM solution in {{elastic-sec}}.
```

Comment:
Our [style guide](https://docs.elastic.dev/tech-writing-guidelines/grammar-spelling#capitalization) suggests capitalizing the first word of sentences (not sure if we include incomplete sentences), so recommending that here as well.

```suggestion
* [Automatic Import](/solutions/security/get-started/automatic-import.md): Helps you quickly parse, ingest, and create [ECS mappings](https://www.elastic.co/elasticsearch/common-schema) for data from sources that don’t yet have prebuilt Elastic integrations. This can accelerate your migration to {{elastic-sec}}, and help you quickly add new data sources to an existing SIEM solution in {{elastic-sec}}.
```